ui/admin/app/router.js

```diff
@@ -27,6 +27,9 @@ Router.map(function () {
         this.route('new');
       });
       this.route('edit');
+      this.route('add-storage-policy', function () {
+        this.route('create');
```

Comment:
None blocking comment: most of the routes, use `new` rather than `create`. I have no argument about it, but I would liek to know why we choose `create`.

Reply:
I guess this is not a traditional `new` that you see in other routes, this is a new sub route within the main., similar to a storage bucket creation within target

Reply:
@DhariniJeeva ah gotcha!! that makes total sense. Thanks for the clarification :wink: